CRITICAL - 2022-01-29 00:05:38 --> Syntax error, malformed JSON
#0 E:\xampp\htdocs\lab\vendor\firebase\php-jwt\src\JWT.php(319): Firebase\JWT\JWT::handleJsonError(4)
#1 E:\xampp\htdocs\lab\vendor\firebase\php-jwt\src\JWT.php(98): Firebase\JWT\JWT::jsonDecode('\xC8\x9D\x1E\\\x08\x8E\x88\x92\x95\xD5\x08\x8B\x08\x98[...')
#2 E:\xampp\htdocs\lab\app\Filters\Auth.php(64): Firebase\JWT\JWT::decode('yJ0eXAiOiJKV1Qi...', 'cskhfuwt48wbfjn...', Array)
#3 E:\xampp\htdocs\lab\system\Filters\Filters.php(173): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#4 E:\xampp\htdocs\lab\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('me', 'before')
#5 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-01-29 00:05:38 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in E:\xampp\htdocs\lab\system\Format\JSONFormatter.php:41
Stack trace:
#0 E:\xampp\htdocs\lab\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 E:\xampp\htdocs\lab\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 E:\xampp\htdocs\lab\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 E:\xampp\htdocs\lab\system\Debug\Exceptions.php(115): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(DomainException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-01-29 00:06:18 --> $keyOrKeyArray must be an instance of Firebase\JWT\Key key or an array of Firebase\JWT\Key keys
#0 E:\xampp\htdocs\lab\vendor\firebase\php-jwt\src\JWT.php(114): Firebase\JWT\JWT::getKey('cskhfuwt48wbfjn...', NULL)
#1 E:\xampp\htdocs\lab\app\Filters\Auth.php(64): Firebase\JWT\JWT::decode('eyJ0eXAiOiJKV1Q...', 'cskhfuwt48wbfjn...', Array)
#2 E:\xampp\htdocs\lab\system\Filters\Filters.php(173): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 E:\xampp\htdocs\lab\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('me', 'before')
#4 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-29 00:07:07 --> $keyOrKeyArray must be an instance of Firebase\JWT\Key key or an array of Firebase\JWT\Key keys
#0 E:\xampp\htdocs\lab\vendor\firebase\php-jwt\src\JWT.php(114): Firebase\JWT\JWT::getKey('cskhfuwt48wbfjn...', NULL)
#1 E:\xampp\htdocs\lab\app\Filters\Auth.php(64): Firebase\JWT\JWT::decode('eyJ0eXAiOiJKV1Q...', 'cskhfuwt48wbfjn...', Array)
#2 E:\xampp\htdocs\lab\system\Filters\Filters.php(173): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 E:\xampp\htdocs\lab\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('me', 'before')
#4 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-29 00:08:25 --> Class 'App\Filters\Key' not found
#0 E:\xampp\htdocs\lab\system\Filters\Filters.php(173): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#1 E:\xampp\htdocs\lab\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('me', 'before')
#2 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-01-29 00:09:08 --> Syntax error, malformed JSON
#0 E:\xampp\htdocs\lab\vendor\firebase\php-jwt\src\JWT.php(319): Firebase\JWT\JWT::handleJsonError(4)
#1 E:\xampp\htdocs\lab\vendor\firebase\php-jwt\src\JWT.php(98): Firebase\JWT\JWT::jsonDecode('\xC8\x9D\x1E\\\x08\x8E\x88\x92\x95\xD5\x08\x8B\x08\x98[...')
#2 E:\xampp\htdocs\lab\app\Filters\Auth.php(66): Firebase\JWT\JWT::decode('yJ0eXAiOiJKV1Qi...', Object(Firebase\JWT\Key))
#3 E:\xampp\htdocs\lab\system\Filters\Filters.php(173): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#4 E:\xampp\htdocs\lab\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('me', 'before')
#5 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-01-29 00:09:08 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in E:\xampp\htdocs\lab\system\Format\JSONFormatter.php:41
Stack trace:
#0 E:\xampp\htdocs\lab\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 E:\xampp\htdocs\lab\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 E:\xampp\htdocs\lab\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 E:\xampp\htdocs\lab\system\Debug\Exceptions.php(115): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(DomainException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-01-29 00:09:30 --> Signature verification failed
#0 E:\xampp\htdocs\lab\app\Filters\Auth.php(66): Firebase\JWT\JWT::decode('eyJ0eXAiOiJKV1Q...', Object(Firebase\JWT\Key))
#1 E:\xampp\htdocs\lab\system\Filters\Filters.php(173): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#2 E:\xampp\htdocs\lab\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('me', 'before')
#3 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-29 00:53:06 --> Expired token
#0 E:\xampp\htdocs\lab\app\Filters\Auth.php(66): Firebase\JWT\JWT::decode('eyJ0eXAiOiJKV1Q...', Object(Firebase\JWT\Key))
#1 E:\xampp\htdocs\lab\system\Filters\Filters.php(173): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#2 E:\xampp\htdocs\lab\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('register', 'before')
#3 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-29 01:32:34 --> Declaration of App\Controllers\Page::create(CodeIgniter\HTTP\RequestInterface $request) should be compatible with CodeIgniter\RESTful\ResourceController::create()
#0 E:\xampp\htdocs\lab\app\Controllers\Page.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Declaration of ...', 'E:\\xampp\\htdocs...', 13, Array)
#1 E:\xampp\htdocs\lab\system\Router\Router.php(518): include_once('E:\\xampp\\htdocs...')
#2 E:\xampp\htdocs\lab\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('page')
#3 E:\xampp\htdocs\lab\system\CodeIgniter.php(719): CodeIgniter\Router\Router->handle('page')
#4 E:\xampp\htdocs\lab\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#5 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-01-29 01:44:01 --> Cannot use object of type stdClass as array
#0 E:\xampp\htdocs\lab\system\CodeIgniter.php(825): App\Controllers\Page->create()
#1 E:\xampp\htdocs\lab\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Page))
#2 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-01-29 02:01:12 --> Class 'App\Models\PostModel' not found
#0 E:\xampp\htdocs\lab\system\CodeIgniter.php(825): App\Controllers\Post->index()
#1 E:\xampp\htdocs\lab\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Post))
#2 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-01-29 02:01:26 --> Expired token
#0 E:\xampp\htdocs\lab\app\Filters\Auth.php(52): Firebase\JWT\JWT::decode('eyJ0eXAiOiJKV1Q...', Object(Firebase\JWT\Key))
#1 E:\xampp\htdocs\lab\system\Filters\Filters.php(173): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#2 E:\xampp\htdocs\lab\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('post', 'before')
#3 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2022-01-29 02:07:20 --> Column 'page_id' cannot be null
CRITICAL - 2022-01-29 02:07:20 --> Column 'page_id' cannot be null
#0 E:\xampp\htdocs\lab\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `po...', 0)
#1 E:\xampp\htdocs\lab\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `po...')
#2 E:\xampp\htdocs\lab\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `po...')
#3 E:\xampp\htdocs\lab\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#4 E:\xampp\htdocs\lab\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\xampp\htdocs\lab\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 E:\xampp\htdocs\lab\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\xampp\htdocs\lab\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 E:\xampp\htdocs\lab\app\Controllers\Post.php(85): CodeIgniter\BaseModel->save(Array)
#9 E:\xampp\htdocs\lab\system\CodeIgniter.php(825): App\Controllers\Post->create()
#10 E:\xampp\htdocs\lab\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Post))
#11 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-01-29 02:17:45 --> Column 'post_content' cannot be null
CRITICAL - 2022-01-29 02:17:45 --> Column 'post_content' cannot be null
#0 E:\xampp\htdocs\lab\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `po...', 0)
#1 E:\xampp\htdocs\lab\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `po...')
#2 E:\xampp\htdocs\lab\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `po...')
#3 E:\xampp\htdocs\lab\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#4 E:\xampp\htdocs\lab\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\xampp\htdocs\lab\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 E:\xampp\htdocs\lab\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\xampp\htdocs\lab\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 E:\xampp\htdocs\lab\app\Controllers\Post.php(84): CodeIgniter\BaseModel->save(Array)
#9 E:\xampp\htdocs\lab\system\CodeIgniter.php(825): App\Controllers\Post->create('2')
#10 E:\xampp\htdocs\lab\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Post))
#11 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2022-01-29 03:41:48 --> Undefined variable: db
#0 E:\xampp\htdocs\lab\app\Controllers\Feed.php(49): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'E:\\xampp\\htdocs...', 49, Array)
#1 E:\xampp\htdocs\lab\system\CodeIgniter.php(825): App\Controllers\Feed->index()
#2 E:\xampp\htdocs\lab\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Feed))
#3 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-29 03:42:10 --> Undefined property: App\Controllers\Feed::$db
#0 E:\xampp\htdocs\lab\app\Controllers\Feed.php(49): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'E:\\xampp\\htdocs...', 49, Array)
#1 E:\xampp\htdocs\lab\system\CodeIgniter.php(825): App\Controllers\Feed->index()
#2 E:\xampp\htdocs\lab\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Feed))
#3 E:\xampp\htdocs\lab\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 E:\xampp\htdocs\lab\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
